cinema_service/settings.py

```diff
@@ -121,12 +119,11 @@
 
 LANGUAGE_CODE = "en-us"
 
-TIME_ZONE = "UTC"
+TIME_ZONE = "Europe/Kiev"
```

Comment:
The `TIME_ZONE` is set to `Europe/Kiev`. Ensure that this is the desired time zone for your application, or consider using `UTC` for consistency across different regions.